utils/process-stats-dir.py

```diff
@@ -674,31 +700,8 @@ def main():
                         help="stats-dirs to process")
 
     args = parser.parse_args()
-    if len(args.remainder) == 0:
-        parser.print_help()
-        return 1
-    if args.catapult:
-        write_catapult_trace(args)
-    elif args.compare_stats_dirs:
-        return compare_stats_dirs(args)
-    elif args.set_csv_baseline is not None:
-        return set_csv_baseline(args)
-    elif args.compare_to_csv_baseline is not None:
-        return compare_to_csv_baseline(args)
-    elif args.incrementality:
-        if args.paired:
-            show_paired_incrementality(args)
-        else:
-            show_incrementality(args)
-    elif args.lnt:
-        write_lnt_values(args)
-    elif args.evaluate:
-        return evaluate(args)
-    elif args.evaluate_delta:
-        return evaluate_delta(args)
-    elif args.render_profiles:
-        return render_profiles(args)
-    return None
-
+    r = process(args)
+    args.output.close()
+    return r
```

Comment:
optional nitpick: possibly this may need to be wrapped in a `try`/`finally` or a `with` statement, but this is probably overkill for the scope of this program

Reply:
Ah yeah, good suggestion!